test/testdata/rewriter/concern.rb.rewrite-tree.exp

```diff
@@ -0,0 +1,192 @@
+class <emptyTree><<C <root>>> < (::<todo sym>)
+  module <emptyTree>::<C ActiveSupport>::<C Concern><<C <todo sym>>> < ()
+  end
+
+  module <emptyTree>::<C Foo><<C <todo sym>>> < ()
+    def instance_method<<C <todo sym>>>(&<blk>)
+      <emptyTree>
+    end
+
+    <self>.extend(<emptyTree>::<C ActiveSupport>::<C Concern>)
+
+    <self>.extend(::T::Helpers)
+
+    ::Sorbet::Private::Static.keep_def(<self>, :"instance_method")
+
+    <self>.mixes_in_class_methods(<emptyTree>::<C ClassMethods>)
```

Comment:
This references a constant before it's defined. Can we move this to after the definition of `module ClassMethods`?

Reply:
Done. 

Ordering of the nodes is not preserved perfectly compared to the input due to the implementation itself. I didn't think it was necessary and I think it would result in a more complicated logic to achieve it. I did the minimal change to reference the constant after its defined.